packages/api-client-core/src/InternalModelManager.ts

```diff
@@ -484,20 +484,21 @@ export class InternalModelManager<Shape extends RecordShape = RecordData> {
    * // upserts post with id 10 in the database
    * // if a post with id 10 exists, it will be updated
    * // if a post with id 10 does not exist, it will be created
-   * const post = await api.internal.post.upsert({ id: "10", title: "A new post title" });
+   * const post = await api.internal.post.upsert({ post: {id: "10", title: "A new post title"} });
```

Comment:
I think it might make sense to document the simple form cause that form will work, but then show this one below?

Reply:
👍